docs/learn/bitesize/bitesize-socialhub.md

```diff
@@ -1,30 +1,32 @@
 ---
-id: bitesize-usecases
-title: Multiplayer Use Cases sample
+id: bitesize-socialhub
+title: Multiplayer Distributed Authority Social Hub sample
 description: Learn about the scenes in the Multiplayer Use Cases sample.
 ---
 
-The [Distributed Authority Social Hub Sample](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize/tree/main/Basic/DistributedAuthoritySocialHub) is a project aimed to demonstrate Distributed Authority's core features and benefits, helping users seamlessly integrate Distributed Authority into their own game projects.
-Within Social Hub, you are free to explore Distributed Authority's key features & functionalities in a sandbox-like environment. Such features include:
+The [Distributed Authority Social Hub Sample](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize/tree/main/Basic/DistributedAuthoritySocialHub) is a project that demonstrates Distributed Authority's features and helps you integrate Distributed Authority into your own game projects.
 
-* Automatic redistribution of ownership of NetworkObjects across connected clients
-* Client-side NetworkObject spawning and deferred NetworkObject despawning
-* Ownership transfer of NetworkObjects with variable ownership flags
-* Host migration of a networked game session
+Within Social Hub, you can explore Distributed Authority in a sandbox-like environment. You can:
+
+- Automatically redistribute ownership of NetworkObjects across connected clients
+- Client-side NetworkObject spawning and deferred NetworkObject despawning
+- Transfer ownership of NetworkObjects with variable ownership flags
+- Host migration of a networked game session
 
 ## The Bootstrap scene
 
-This is the entry point to the sample. Once loaded, you will be transitioned to the MainMenu scene.
+This is the entry point to the sample. After it loads, the MainMenu scene appears.
 
 ## The MainMenu scene
```

Comment:
What is the main task the user will accomplish with this?
Side note: I could not get this to load right for me.

Reply:
A successful load would mean that they've correctly registered their project with a cloud project id.